### DISCUSSION TITLE
Tanya error

### DISCUSSION THREAD

**Discussion by AjiPramasto** [teamalgoritma/community#245] [Q&A] [answered]
Halo

Mohon bantuannya, untuk code ini sebelumnya bisa dijalankan. sekarang menjadi error:
![image](https://user-images.githubusercontent.com/107389614/182571047-23abfab4-0d36-4692-960b-9706330c11a0.png)

Terima kasih

**Comment by kevbow** [comment, marked as answer]
Halo @AjiPramasto,

Apakah Anda menjalankan chunk ini sebanyak dua kali secara berurutan?

**Reply by AjiPramasto** [reply]
tidak Kak

**Reply by kevbow** [reply]
Bisa coba difotokan kalau begitu pak untuk isi pada dataframe `data`?

**Reply by AjiPramasto** [reply]
berikut ya Ka
![image](https://user-images.githubusercontent.com/107389614/182574208-0dba243a-a571-499a-a725-c535d9bf01b9.png)

**Reply by kevbow** [reply]
Seharusnya tidak ada masalah dari kode yang dibuat, bisa coba untuk di clear objects from the workspace menggunakan tanda sapu di sebelah kanan atas lalu terminate R dan jalankan ulang kode Anda dari awal.
Untuk errornya dapat mengacu infonya pada [link ini](https://statisticsglobe.com/error-replacement-has-x-rows-data-has-y-in-r).
